### PR TITLE
Add validation for Kind and APIVersion

### PR DESCRIPTION
**Pull request by josefkarasek**
Signed-off-by: Josef Karasek <jkarasek@redhat.com>

### CODE REVIEW

File: pkg/validation/internal/csv.go [hunk 1]
```diff
@@ -194,3 +196,15 @@ func validateInstallModes(csv *v1alpha1.ClusterServiceVersion) (errs []errors.Er
 	}
 	return errs
 }
+
+// validateVersionKind checks presense of GroupVersionKind.Verion and GroupVersionKind.Kind
```
Comment (estroz) [inline pkg/validation/internal/csv.go:200]:
```suggestion
// validateVersionKind checks presence of GroupVersionKind.Version and GroupVersionKind.Kind
```

**Review by estroz — COMMENTED** [review]
any reason to not ensure group is present?

**Comment by josefkarasek** [comment]
> any reason to not ensure group is present?

`group` is rarely present

Sometimes we can find something like `apiVersion: operators.coreos.com/v1alpha1`

**Pull request merged**